Makefile

```diff
@@ -224,3 +224,7 @@ kubelet:
 .PHONY: kube-proxy
 kube-proxy:
 	KUBE_GIT_VERSION=$(KUBE-PROXY_GIT_VERSION) KUBE_BUILD_PLATFORMS=windows/amd64 make -C kube-proxy WHAT=cmd/kube-proxy
+
+.PHONY : containerd ## Ref: Containerd Version https://github.com/containerd/containerd/blob/8cb00f45c973b017f403d05d4d208ab2971ff335/Makefile#L33
+containerd:
+	GOOS=windows VERSION=$(shell cd containerd && git describe --match 'v[0-9]*' --dirty='.m' --always) make -C containerd bin/containerd.exe
```

Comment:
Does this still require tags to be manually applied to the downstream branch?

Reply:
yes I think we still need to have tags downstream which are in sync with upstream tags, since this command does a describe on the pattern matching the current tag in the submodule directory.

Reply:
Could we then do something similar like we did for the k8s submodules? i.e. pull the tags when updating the submodules and hard code the version here.